lib/types.d.ts

```diff
@@ -3,6 +3,9 @@ export interface Config {
     path: string;
     themeName: string;
   };
+  prettier?: {
+    path: string;
```

Comment:
Hm… this sounds like it’s the “path to Prettier (the executable)”, not necessarily the config path. Maybe rename this option to `configPath` or something similar?